clouddriver-loadtest/src/main/scala/com/netflix/spinnaker/orca/loadtest/ClouddriverSimulationEngine.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2017 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.spinnaker.orca.loadtest
+
+import io.gatling.app.Gatling
+import io.gatling.core.config.GatlingPropertiesBuilder
+
+object ClouddriverSimulationEngine extends App {
+  val props = new GatlingPropertiesBuilder
+  props.simulationClass(classOf[ClouddriverSimulation].getName)
```

Comment:
@ajordens @robzienert Gatling will read the`gatling.conf` in the `resources`  or `conf` directory.  At runtime you can also pass in Gatling configuration via system properties.